config/initializers/action_mailer.rb

```diff
@@ -2,10 +2,10 @@
   address: ENV['SMTP_SERVER'] || "smtp.gmail.com",
   port: ENV['SMTP_PORT'] || 587,
   domain: ENV['SMTP_DOMAIN'],
-  authentication: ENV['SMTP_AUTHENTICATION'] || "plain",
+  authentication: ENV['SMTP_AUTHENTICATION'] == 'none' ? nil : ENV['SMTP_AUTHENTICATION'] || "plain",
```

Comment:
It's only on `ENV['SMTP_AUTHENTICATION']` otherwise the change would not have worked for @ashleysommer.